src/Strategy/Carthage.hs

```diff
@@ -136,7 +136,7 @@ entryToCheckoutName entry =
   case resolvedType entry of
     GitEntry -> resolvedName entry
     -- this is safe because Text.splitOn always returns a non-empty list
-    GithubType -> last . Text.splitOn "/" $ resolvedName entry
+    GithubType -> snd . Text.splitOnceOnEnd "/" $ resolvedName entry
```

Comment:
cc @cnr

Reply:
should kill the "this is safe because..." comments on each of these `Text.splitOn` changes